module/VuFind/src/VuFind/Content/TOC/ObalkyKnih.php

```diff
@@ -0,0 +1,80 @@
+<?php
+
+/**
+ * Obalky knih TOC content loader.
+ *
+ * PHP version 8
+ *
+ * Copyright (C) Moravian Library 2024.
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 2,
+ * as published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
+ *
+ * @category VuFind
+ * @package  Content
+ * @author   Josef Moravec <moravec@mzk.cz>
+ * @license  http://opensource.org/licenses/gpl-2.0.php GNU General Public License
+ * @link     https://vufind.org/wiki/development Wiki
+ */
+
+declare(strict_types=1);
+
+namespace VuFind\Content\TOC;
+
+use VuFind\Content\ObalkyKnihService;
+
+/**
+ * Class ObalkyKnih
+ *
+ * @category VuFind
+ * @package  Content
+ * @author   Josef Moravec <moravec@mzk.cz>
+ * @license  https://opensource.org/licenses/gpl-2.0.php GNU General Public License
+ * @link     https://vufind.org/wiki/development Wiki
+ */
+class ObalkyKnih extends \VuFind\Content\AbstractBase
+{
+    /**
+     * Constructor
+     *
+     * @param ObalkyKnihService $service Service for getting metadata from obalkyknih.cz
+     */
+    public function __construct(protected ObalkyKnihService $service)
+    {
+    }
+
+    /**
+     * This method is responsible for generating fake TOC data for testing
```

Comment:
I think this comment is inaccurate -- probably copied from the demo code.

Reply:
Fixed.